src/Tickets/Commerce/Admin_Tables/Orders_Table.php

```diff
@@ -83,7 +83,7 @@ public function search_box( $text, $input_id ) {
 
 		$text = __( 'Search Orders', 'event-tickets' );
 		?>
-			<p class="search-box">
+			<div class="search-box">
```

Comment:
`<p>` replacement to `<div>` without amending closing tag

Reply:
🤦🏼